src/Builder/ConfigurationEnvironmentBuilder/ArrayConfigurationEnvironmentStrategy.php

```diff
@@ -45,13 +45,11 @@ protected function createExpressionStringFromValue($value, int $level = 0): stri
         $result = ['['];
 
         foreach ($value as $keyValueItem => $valueItem) {
-            $outputFormat = '\'%s\',';
             if (is_array($valueItem)) {
                 $valueItem = $this->createExpressionStringFromValue($valueItem, $level);
-                $outputFormat = '%s,';
             }
             if (is_int($keyValueItem)) {
-                $result[] = $this->createIndent($level) . sprintf($outputFormat, trim($valueItem, '\''));
+                $result[] = $this->createIndent($level) . sprintf('%s,', $this->getFormattedValueExpression($valueItem));
```

Comment:
Just for the future: I think it is a standard naming convention that 'get' prefix should be used only for getters - methods, that don't contain any logic, just return the value of the object attribute. 

In this case if method calculates something, it's better to name it 'calculate...' instead of 'getCalculated...', if it formats something => 'format...' instead of 'getFormatted...'.

But this method didn't come in this PR, so it is not new, so there is no need to change it now. We can rename it in one of the future PRs.